x-pack/plugins/maps/public/classes/sources/source.ts

```diff
@@ -41,6 +39,7 @@ export interface ISource {
   getDisplayName(): Promise<string>;
   getInspectorAdapters(): Adapters | undefined;
   isFieldAware(): boolean;
+  isEditable(): Promise<boolean>;
```

Comment:
Same comment as with ILayer vs IVectorLayer. Would it be possible to move method to IVectorSource interface?